packages/server/lib/browsers/chrome.coffee

```diff
@@ -315,18 +325,24 @@ module.exports = {
         .then (criClient) =>
           la(criClient, "expected Chrome remote interface reference", criClient)
 
-          ## monkey-patch the .kill method to that the CDP connection is closed
-          originalBrowserKill = launchedBrowser.kill
+          criClient.ensureMinimumProtocolVersion('1.3')
```

Comment:
there are no tests verifying this function is ever called. there's tests on the actual `ensureMinimumProtocolVersion` function, but no tests that `_writeExtension` ever calls this.

There's likely existing `integration/cypress_spec` tests that test things very close to this - or you could write a unit tests for `browsers/chrome`

Reply:
There is a test that the error is thrown here: https://github.com/cypress-io/cypress/blob/b2825a6936d1b6fe86a46b30b3d783b5d634471e/packages/server/test/unit/browsers/chrome_spec.coffee#L142-L145

Added an assertion that @criClient.ensureMinimumProtocol version was called in every #open call: a4b8167bb